src/bench.c

```diff
@@ -11,6 +11,31 @@
 #include "util.h"
 #include "bench.h"
 
+void help(char **argv, int default_iters) {
+    printf("Benchmarks the following algorithms:\n");
+    printf("    1. ECDSA signing/verification\n");
+    printf("    2. ECDH key exchange (optional module)\n");
+    printf("    3. Public key recovery (optional module)\n");
+    printf("    4. Schnorr signatures (optional module)\n");
+    printf("\n");
+    printf("The default number of iterations for each benchmark is %d. This can be\n", default_iters);
+    printf("customized using the SECP256K1_BENCH_ITERS environment variable.\n");
+    printf("\n");
+    printf("Usage: %s [args]\n", argv[0]);
+    printf("By default, all benchmarks will be run.\n");
+    printf("args:\n");
+    printf("    help              : display this help and exit\n");
+    printf("    ecdsa             : all ecdsa algorithms (sign, verify, recover)\n");
+    printf("    ecdsa_sign        : ecdsa siging algorithm\n");
+    printf("    ecdsa_verify      : ecdsa verification algorithm\n");
+    printf("    ecdsa_recover     : ecdsa public key recovery algorithm\n");
+    printf("    ecdh              : ecdh key exchange algorithm\n");
+    printf("    schnorrsig        : all schnorr signature algorithms (sign, verify)\n");
+    printf("    schnorrsig_sign   : schnorr sigining algorithm\n");
+    printf("    schnorrsig_verify : schnorr verification algorithm\n");
```

Comment:
nit: Could capitalize ECDSA, ECDH and Schnorr here in the right column if you're on it.

Reply:
Sure!